sdk/core/Azure.Core/src/Shared/NoValueResponseOfT.cs

```diff
@@ -2,18 +2,17 @@
 // Licensed under the MIT License.
 
 using System;
+using Azure.Core;
 
 namespace Azure
 {
 #pragma warning disable SA1649 // File name should match first type name
     internal sealed class NoValueResponse<T> : NullableResponse<T>
```

Comment:
This breaks because it is shared source and some projects aren't referencing Azure.Core 2.0.

Filed https://github.com/Azure/azure-sdk-for-net/issues/40557 to track doing the work as time allows.